src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/dynamicLists/DynamicRespondentRepresentative.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers.dynamicLists;
+
+import uk.gov.hmcts.ecm.common.model.bulk.types.DynamicFixedListType;
+import uk.gov.hmcts.ecm.common.model.bulk.types.DynamicValueType;
+import uk.gov.hmcts.ecm.common.model.ccd.CaseData;
+import uk.gov.hmcts.ecm.common.model.ccd.items.RepresentedTypeRItem;
+import uk.gov.hmcts.ecm.common.model.ccd.types.RepresentedTypeR;
+import uk.gov.hmcts.ethos.replacement.docmosis.helpers.DynamicListHelper;
+
+import java.util.List;
+import java.util.ListIterator;
+
+public class DynamicRespondentRepresentative {
+
+    public static void dynamicRespondentRepresentativeNames(CaseData caseData) {
+        List<DynamicValueType> listItems = DynamicListHelper.createDynamicRespondentName(caseData.getRespondentCollection());
+        if (!listItems.isEmpty()) {
+            var dynamicFixedListType = new DynamicFixedListType();
+            dynamicFixedListType.setListItems(listItems);
+            if (caseData.getRepCollection() != null && !caseData.getRepCollection().isEmpty()) {
```

Comment:
using CollectionUtils.isNotEmpty(caseData.getRepCollection()) instead here for both null & empty collection check?